identity/webapp/src/frontend/MyAccount/ChangeEmail.tsx

```diff
@@ -37,6 +38,10 @@ export const ChangeEmail: React.FC<ChangeDetailsModalContentProps> = ({ onComple
         setError('password', { type: 'manual', message: 'Incorrect password.' });
         break;
       }
+      case RequestDeleteError.BRUTE_FORCE_BLOCKED: {
+        setError('password', { type: 'manual', message: 'Your account has been blocked after multiple consecutive login attempts.' });
```

Comment:
This would highlight the `password` field, but this error isn't necessarily caused by a user error in the password field. Would it be better to show this error in a way that applies to the whole form?

```suggestion
        setSubmissionErrorMessage('Your account has been blocked after multiple consecutive login attempts.');
```

...but since the user is now unable to perform any action and is potentially a bad actor looking at an account that isn't theirs, is it better to log them out at this point and show them an explanatory message?